src/Integration.Vsix.UnitTests/CFamily/MessageHandlerTests.cs

```diff
@@ -186,7 +250,53 @@ private static IRequest CreateRequest(string file = null, string language = null
             return request.Object;
         }
 
-        private static Message CreateMessage(string ruleId, string fileName = "any file") =>
-            new Message(ruleId, fileName, -1, -1, -1, -1, "any text", false, null);
+        private static Message CreateMessage(string ruleId, string fileName = "any file", string text = "any text") =>
+            new Message(ruleId, fileName, -1, -1, -1, -1, text, false, null);
+
+
+        private class MessageHandlerTestContext
```

Comment:
New helper class to simplify setting up the tests. I'll simplify the existing tests to use the helper class separately.